### PR TITLE
Rename the helper method to its final form - "GetPinnableReference"

### PR DESCRIPTION
**Pull request by VSadov**

### CODE REVIEW

**Comment by VSadov** [comment]
@dotnet/roslyn-compiler - please review. It is a trivial change. Mostly renames due to the change of the name of the special method.

File: src/Compilers/CSharp/Portable/Binder/Binder_Statements.cs [hunk 1]
```diff
@@ -1057,7 +1057,7 @@ private bool IsValidFixedVariableInitializer(TypeSymbol declType, SourceLocalSym
 
             TypeSymbol elementType;
             bool hasErrors = false;
-            MethodSymbol getPinnableMethod = null;
+            MethodSymbol fixedpatternMethod = null;
```
Comment (cston) [inline src/Compilers/CSharp/Portable/Binder/Binder_Statements.cs]: `fixedPatternMethod`

Comment (VSadov) [reply src/Compilers/CSharp/Portable/Binder/Binder_Statements.cs]: fixed, thanks!

---
In reply to: [170078300](https://github.com/dotnet/roslyn/pull/24981#discussion_r170078300) [](ancestors = 170078300)

**Comment by VSadov** [comment]
Thanks!!

**Pull request merged**